aspnetcore/mvc/models/validation.md

```diff
@@ -17,6 +17,8 @@ Before an app stores data in a database, the app must validate the data. Data mu
 
 Fortunately, .NET has abstracted validation into validation attributes. These attributes contain validation code, thereby reducing the amount of code you must write.
 
+In ASP.NET Core 2.2 preview 3 and later, the ASP.NET Core runtime short-circuits (skips) validation if it can determine that a given model graph doesn't require validation. Skipping validation can provide significant performance improvements when validating models that cannot or do not have any associated validators. The skipped validation includes objects such as collections of primitives (`byte[]`, `string[]`, `Dictionary<string, string>`, etc.), or complex object graphs without any validators.
```

Comment:
Can the "preview3" part of this text be removed? That will make it more future-proof.